x-pack/plugins/security_solution/cypress/e2e/detection_alerts/changing_alert_status.cy.ts

```diff
@@ -240,30 +221,25 @@ describe('Changing alert status', () => {
 
           const expectedNumberOfAlerts = +numberOfAlerts - numberOfAlertsToBeClosed;
           cy.get(ALERTS_COUNT).should('have.text', `${expectedNumberOfAlerts} alerts`);
-          cy.get(ALERT_COUNT_TABLE_FIRST_ROW_COUNT).should(
-            'have.text',
-            `${expectedNumberOfAlerts}`
-          );
+          checkAlertCountFromAlertCountTable(expectedNumberOfAlerts);
 
           goToClosedAlerts();
           waitForAlerts();
 
           cy.get(ALERTS_COUNT).should('have.text', `${numberOfAlertsToBeClosed} alert`);
-          cy.get(ALERT_COUNT_TABLE_FIRST_ROW_COUNT).should(
-            'have.text',
-            `${numberOfAlertsToBeClosed}`
-          );
+
+          checkAlertCountFromAlertCountTable(numberOfAlertsToBeClosed);
         });
     });
 
-    it('Updates trend histogram whenever alert status is updated in table', () => {
+    it('Updates count table whenever alert status is updated in table', () => {
```

Comment:
@MadameSheema , I changed the test description based on the content below, but I might have misunderstood, could you please check if I took it right?